src/benchmarks/gc/GC.Infrastructure/GC.Analysis.API/AnalyzerManager.cs

```diff
@@ -22,7 +22,7 @@ public static class AnalyzerManager
         {
             Dictionary<string, Analyzer> analyzers = new();
 
-            IEnumerable<string> allFiles = null;
+            IEnumerable<string> allFiles;
 
```

Comment:
I presume this is to fix a warning?

Reply:
Yes, it's not `IEnumerable<string>?`, so `null` isn't appropriate.  The original value isn't needed since the `if/else` takes care of it.  This is from diving into the nullable warnings, which is a much larger project but I kept the few things that I did.